includes/derivatives.inc

```diff
@@ -296,72 +320,37 @@ function islandora_ocr_derive_ocr(AbstractObject $object, $force = FALSE) {
       );
     }
     file_unmanaged_delete($file);
+    call_user_func_array("islandora_ocr_remove_generate_{$lc_type}_rels_ext_statement", array($object));
```

Comment:
In the case of the objects created before the update date wouldn't be hit here... The relationship would be removed when processing the OCR, so it wouldn't (/shouldn't) be [available to start the HOCR stuff](https://github.com/Islandora/islandora_ocr/pull/81/commits/a6b45ed4f56a94384079c0fc7c2b5508d4051115#diff-f20cd9f6981182cb8c80727774d28d4fR350).